src/burn/engine/variable.cpp

```diff
@@ -1897,26 +1897,35 @@ static HRESULT InitializeVariableSystemFolder(
     BOOL f64 = (BOOL)dwpData;
     WCHAR wzSystemFolder[MAX_PATH] = { };
 
-#ifndef _WIN64
```

Comment:
I'm fine if we want to say this needs to move to WiX v4.0 because it is "a fix breaking today's behavior". Adding warning doesn't make sense to me.